go/dhcp/pool/pool.go

```diff
@@ -6,6 +6,9 @@ import (
 	"sync"
 )
 
+const FreeMac = "00:00:00:00:00:00"
```

Comment:
Create a package for constants and centralize them there so they aren't duplicated in each package